revscores/languages/portuguese.py

```diff
@@ -1064,21 +1064,20 @@
     "zueira"
 ])
 
-class Portuguese(Language):
+def is_badword(word):
+    return STEMMER.stem(word).lower() in BADWORDS
```

Comment:
Considering that the definition of `is_badword` is the same for Portuguese and English, isn't possible to make this to be a "default implementation" so that we don't need to have two (or more) copies of it?